secure/key-manager/Cargo.toml

```diff
@@ -35,7 +35,9 @@ tokio = { version = "0.2.13", features = ["full"] }
 config-builder = { path = "../../config/config-builder", version = "0.1.0" }
 executor = { path = "../../execution/executor", version = "0.1.0" }
 executor-types = { path = "../../execution/executor-types", version = "0.1.0" }
-libradb = { path = "../../storage/libradb", version = "0.1.0" }
+libra-config = { path = "../../config", version = "0.1.0", features = ["fuzzing"]}
```

Comment:
move this to line 45/47

libra-config/fuzzing in the fuzzing line

same thing with libra-secure-storage

no idea how this leaked into our dependency list :(

Reply:
All fuzzing/testing feature crates at the bottom of the list, or be alphabetical?   trying to figure out exactly what you want here?

Reply:
ahh inlined in the fuzzing feature

Reply:
done.